app/src/main/java/com/yeonae/chamelezone/data/model/PlaceItem.kt

```diff
@@ -0,0 +1,11 @@
+package com.yeonae.chamelezone.data.model
+
+import java.io.Serializable
+
+data class PlaceItem @JvmOverloads constructor(
```

Comment:
```suggestion
data class PlaceItem(
```
아래의 변수에 오버로딩을 구현하지 않았으므로 필요없는 부분 같습니다.

Reply:
수정했습니당